e2e/cephfs_helper.go

```diff
@@ -66,6 +66,12 @@ func createCephfsStorageClass(
 	if err != nil {
 		return err
 	}
+	// TODO: remove this once the ceph-csi driver release-v3.9 is completed
+	// and upgrade tests are done from v3.9 to devel.
+	// The mountOptions from previous are not compatible with NodeStageVolume
+	// request.
+	sc.MountOptions = []string{}
+
```

Comment:
CephFS upgrade tests pick up cephfs sc from previous release branches which still
have `debug` mountoption in sc yamls. 
Adding this workaround untill we set upgrade test from 3.9 to devel.

I'll add a note regarding this breaking change in release notes too.
(users will need to recreate cephfs storage without debug options and 
remove it from pvs too.)


added issue to track this one https://github.com/ceph/ceph-csi/issues/3911

I'll ask re-review requests after all the ci is green.
(:sweat_smile: this pr seems to be giving more ci issues than expected.)